crates/ibc/src/timestamp.rs

```diff
@@ -26,6 +26,70 @@ pub struct Timestamp {
     time: Option<Time>,
 }
 
+#[cfg(feature = "borsh")]
+impl borsh::BorshSerialize for Timestamp {
+    fn serialize<W: borsh::maybestd::io::Write>(
+        &self,
+        writer: &mut W,
+    ) -> borsh::maybestd::io::Result<()> {
+        let timestamp = if let Some(time) = self.time {
+            time.unix_timestamp_nanos()
+        } else {
+            0i128
```

Comment:
```suggestion
            0
```

Reply:
Can we also add the comment here:

> When the value in `Time` is `None` we give the timestamp a default value of 0

Reply:
Can we address these 2 points too please? @DaviRain-Su

Reply:
done

Reply:
> Can we address these 2 points too please? @DaviRain-Su

Dude, seriously, you're looking really hard.